translations/ethos/de.csv

```diff
@@ -2,89 +2,89 @@ String name, String context, X20(...) / XE(S) / V20, X18RS / X14(...) / T14 / V1
 TRANSLATION_NAME,,"Deutsch"
 TRANSLATION_KEY,,"de"
 FORMULA,,"Formel"
-CALCULATED_SENSOR,,"berechneter Sensor"
+CALCULATED_SENSOR,,"Berechneter Sensor"
 NAME,,"Name"
 DECIMALS,,"Kommastellen"
 VALUE,,"Wert"
 SENSOR_LOST_WARNING_DELAY,,"Sensor verloren, Ansage verz.:"
 WARNING_DISABLED,,"Warnung deaktiviert"
 RANGE,,"Bereich"
 SPACE_WITH_SAMPLES,," Messungen"
-RESET,,"zurücksetz."
-TELEMETRY_SENSOR,,"Telemetrie Sensor"
+RESET,,"zurücksetzen"
```

Comment:
No risk it doesn't fit on small screens? No luck this translation is used in many places in Ethos:
- play audio function
- reset function
- line chart widget
- DIY sensor
- model edit
- model sensor edit
- model telemetry
- model timer
- model timers
- system information
- calculated sensor
- trip sensor